src/components/menu/scrollable-block/scrollable-block.component.js

```diff
@@ -10,6 +11,8 @@ const ScrollableBlock = ({
   children,
   height,
   variant = "default",
+  parent,
```

Comment:
If there's a strong reason behind the naming I'm happy to leave it, but atm I'm not convinced `parent` and `parentVariant` are the best names for these as even though there's now a semantic relationship due to this proposed update the element passed in isn't a parent of the scrollable (box) element. Maybe calling it `firstChild` something along those lines might be more intuitive for consumers 🤷

Reply:
Couldn't we achieve the same thing by wrapping the Search element with MenuItem?. I think that this interface with `parent` prop is slightly confusing tbh